static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I would say that, as an outsider, I would find either as easy to fathom.
Performance - probably negligible, unless someone wants to test it?
Ambivalence = doesn't matter?